test/core/tsi/ssl_transport_security_test.cc

```diff
@@ -360,6 +372,9 @@ static void ssl_test_check_handshaker_peers(tsi_test_fixture* fixture) {
     check_session_reusage(ssl_fixture, &peer);
     check_alpn(ssl_fixture, &peer);
     check_security_level(&peer);
+    if (!ssl_fixture->session_reused) {
```

Comment:
Can we check that this peer property is *not set* for resumed sessions?

Reply:
So you mean in an `else` block after this, explicitly check that there is no peer property with `TSI_X509_CA_SUBJECT_PEER_PROPERTY`?